main-classification.py

```diff
@@ -0,0 +1,405 @@
+                         "SBS15",
+                         "SBS17a",
+                         "SBS17b",
+                         "SBS18",
+                         "SBS31",
+                         "SBS35",
+                         "SBS36",
+                         "SBS37",
+                         "SBS38",
+                         "SBS40",
+                         "SBS44",
+                         "SBS4426",
+                         "ID1",
+                         "ID2",
+                         "ID3",
+                         "ID4",
+                         "ID5",
+                         "ID6",
+                         "ID7",
+                         "ID8",
+                         "ID9",
+                         "ID10",
+                         "ID11",
+                         "ID12",
+                         "ID13",
+                         "ID14",
+                         "ID15",
+                         "ID16",
+                         "ID17",
+                         "ID18"]
+
+    elif sigmut == "comb":
+        features_name = ["Age at advanced disease diagnosis",
+                         "CD8+ T cell score",
+                         "Genome mut per mb",
+                         "Exome mut per mb",
+                         "CD274 expression",
+                         "M1M2 expression",
+                         "SBS1",
+                         "SBS2",
+                         "SBS3",
+                         "SBS4",
+                         "SBS5",
+                         "SBS7a",
+                         "SBS7b",
+                         "SBS7c",
+                         "SBS7d",
+                         "SBS8",
+                         "SBS10a",
+                         "SBS10b",
+                         "SBS10c",
+                         "SBS13",
+                         "SBS15",
+                         "SBS17a",
+                         "SBS17b",
+                         "SBS18",
+                         "SBS31",
+                         "SBS35",
+                         "SBS36",
+                         "SBS37",
+                         "SBS38",
+                         "SBS40",
+                         "SBS44",
+                         "SBS4426",
+                         "ID1",
+                         "ID2",
+                         "ID3",
+                         "ID4",
+                         "ID5",
+                         "ID6",
+                         "ID7",
+                         "ID8",
+                         "ID9",
+                         "ID10",
+                         "ID11",
+                         "ID12",
+                         "ID13",
+                         "ID14",
+                         "ID15",
+                         "ID16",
+                         "ID17",
+                         "ID18"]
+
+    else:
+        raise ValueError("Invalid name of problem. The valid choices are : no-sigmut, only-sigmut, and comb.")
```

Comment:
"Invalid name of problem. The valid choices are : no-sigmut, only-sigmut and comb." (une virgule de plus)